src/main/java/Task.java

```diff
@@ -0,0 +1,26 @@
+public class Task {
+
+    protected String description;
+    protected boolean isDone;
+    public static int taskCount = 0;
```

Comment:
It's possible this might not follow encapsulation: https://nus-cs2113-ay2021s1.github.io/website/schedule/week3/topics.html#paradigms-oop-objects-encapsulation-of-objects
Maybe make it `private`? And add a getter for it as well? The access to the variable in `Duke.java` might need to follow as well.
```suggestion
    private static int taskCount = 0;
    
public static int getTaskCount() {
    return taskCount;
}
```